### PR TITLE
fix build error due to e2e benchmark invalid creating new client

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:
**Please refer to it first!!!** https://github.com/vdaas/vald/pull/1221#issuecomment-866918438

For a long time, the e2e benchmark on the CI ends with failure.
I realized this and search for the point why this job fails.
At last, I found the reason that is because the `grpcAddr` is not set at the core struct.
I fixed it and created this PR.

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: hack/benchmark/e2e/agent/core/ngt/ngt_bench_test.go [hunk 1]
```diff
@@ -51,9 +51,11 @@ func init() {
 func BenchmarkAgentNGT_gRPC_Sequential(b *testing.B) {
 	ctx := context.Background()
 	client, err := core.New(
+		core.WithAddrs(grpcAddr),
 		core.WithGRPCClient(
 			grpc.New(grpc.WithAddrs(grpcAddr),
-				grpc.WithInsecure(true))))
+				grpc.WithInsecure(true))),
+	)
```
Comment (kpango) [inline hack/benchmark/e2e/agent/core/ngt/ngt_bench_test.go:58]:
```suggestion
				grpc.WithInsecure(true))))
```

File: hack/benchmark/e2e/agent/core/ngt/ngt_bench_test.go [hunk 2]
```diff
@@ -86,9 +88,11 @@ func BenchmarkAgentNGT_gRPC_Sequential(b *testing.B) {
 func BenchmarkAgentNGT_gRPC_Stream(b *testing.B) {
 	ctx := context.Background()
 	client, err := core.New(
+		core.WithAddrs(grpcAddr),
 		core.WithGRPCClient(
 			grpc.New(grpc.WithAddrs(grpcAddr),
-				grpc.WithInsecure(true))))
+				grpc.WithInsecure(true))),
+	)
```
Comment (kpango) [inline hack/benchmark/e2e/agent/core/ngt/ngt_bench_test.go:95]:
```suggestion
				grpc.WithInsecure(true))))
```

**Comment by vankichi** [comment]
It seems too chaotic and we should re-write along with the e2e test scenarios.
At this time, we only fix the build error.

**Comment by kpango** [comment]
Since this e2e benchmark only calculates the performance of Agents, I think that a wider range of solutions should be included in the e2e benchmark, including Gateway and others.
So we need to rewrite the entire benchmark framework.

**Review by kpango — APPROVED** [review]
LGTM for now

The build seems to be successful, so let's call it good at this stage.
@vdaas/set Please add this framework issues to the Project milestone as it needs to be rewritten

**Review by hlts2 — APPROVED** [review]
LGTM

**Pull request merged**